### PR TITLE
Edit actions for consistency and add action template.

### PR DESCRIPTION
**Pull request by debadair**
Part of: https://github.com/elastic/elasticsearch/issues/53894

### CODE REVIEW

File: docs/reference/ilm/actions/ilm-readonly.asciidoc [hunk 1]
```diff
@@ -1,17 +1,17 @@
 [role="xpack"]
 [[ilm-readonly]]
-=== Read-only
+=== Read only
 
 Phases allowed: warm.
 
-Sets an index as read-only. See <<dynamic-index-settings,`index.blocks.write`>>.
+Makes the index  <<dynamic-index-settings,`index.blocks.write`, read-only>>.
```
Comment (jrodewig) [inline docs/reference/ilm/actions/ilm-readonly.asciidoc:7]:
```suggestion
Makes the index  <<dynamic-index-settings,read-only>>.
```

File: docs/reference/ilm/actions/ilm-forcemerge.asciidoc [hunk 2]
```diff
@@ -4,22 +4,21 @@
...
-NOTE: The index is made read-only when this action is run.
-See <<dynamic-index-settings,`index.blocks.write`>>.
+<<indices-forcemerge,Force merges>> the index into 
+the specified maximum number of <<indices-segments,segments>>.
 
-NOTE: If the `forcemerge` action is used in the `hot` phase, the `rollover` action *must* be preset.
-{ilm-init} validates this predicate and will refuse a policy with a forcemerge in the hot phase without a
-rollover action.
+This action makes the index read-only.
+See <<dynamic-index-settings,`index.blocks.write`>>.
```
Comment (jrodewig) [inline docs/reference/ilm/actions/ilm-forcemerge.asciidoc:11]: This makes it consistent with the `readonly` action.
```suggestion
This action makes the index <<dynamic-index-settings,read-only>>.
```

File: docs/reference/ilm/actions/ilm-delete.asciidoc [hunk 3]
```diff
@@ -6,16 +6,16 @@ Phases allowed: delete.
...
-==== Delete
+[[ilm-delete-options]]
+==== Options
 
 `delete_searchable_snapshot`::
 (Optional, boolean)
 Deletes the searchable snapshot created in the cold phase. 
 Defaults to `true`.
-This is applicable when the <<ilm-searchable-snapshot-action>> is used in the cold phase.
+This option is applicable when the <<ilm-searchable-snapshot-action>> is used in the cold phase.
```
Comment (jrodewig) [inline docs/reference/ilm/actions/ilm-delete.asciidoc:16]:
```suggestion
This option is applicable when the <<ilm-searchable-snapshot-action,searchable
snapshot>> action is used in the cold phase.
```

File: docs/reference/ilm/actions/_ilm-action-template.asciidoc [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+        "actions": {
+          "sample" : {
+            "sample_option1" : 2
+          }
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
```
Comment (jrodewig) [inline docs/reference/ilm/actions/_ilm-action-template.asciidoc:68]: I believe this is causing the docs-check CI to fail.
We should just skip it.
```suggestion
--------------------------------------------------
// TEST[skip: For illustrative purposes only. Contains fake actions.]
```

File: docs/reference/ilm/actions/_ilm-action-template.asciidoc [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+            "sample_option1" : "100"
+            "sample_option2" : "interesting"
+          }
+        }
+      }
+    }
+  }
+}
+--------------------------------------------------
+
```
Comment (jrodewig) [inline docs/reference/ilm/actions/_ilm-action-template.asciidoc:93]:
```suggestion
// TEST[skip: For illustrative purposes only. Contains fake actions.]
```

**Review by jrodewig — APPROVED** [review]
I added a few suggestions but nothing blocking.

LGTM once the CI passes.

**Comment by debadair** [comment]
@elasticmachine test this please

**Review by andreidan — APPROVED** [review]
LGTM thanks @debadair

**Pull request merged**